lib/plugins/dbsize.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+var _ = require('lodash');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' is assigned a value but never used.](https://app.codacy.com/manual/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)